workspaces/global-header/plugins/global-header/src/components/LogoutButton/LogoutButton.tsx

```diff
@@ -40,7 +40,7 @@ export const LogoutButton = () => {
       onClick={handleLogout}
       sx={{ cursor: 'pointer', width: '100%', color: 'inherit' }}
     >
-      <MenuItemLinkContent icon="logout" label="Logout" />
+      <MenuItemLinkContent icon="logout" label="Sign Out" />
```

Comment:
```suggestion
      <MenuItemLinkContent icon="logout" label="Sign out" />
```

To match the capitalization that GitHub, Google, and the like use